### ISSUE TITLE
Glyph artefacts for more characters [follow-up]

### ISSUE THREAD

**Issue by tonton-pixel** [nomfoundation/font#4]
```
U+23A15 𣨕
U+23E5E 𣹞
U+23F36 𣼶
U+23FB5 𣾵
U+2401E 𤀞
U+2500C 𥀌
U+2512C 𥄬
U+251BA 𥆺
U+2526E 𥉮
U+25CC9 𥳉
U+25CD1 𥳑
U+25D87 𥶇
U+25DFA 𥷺
U+25E13 𥸓
U+2734E 𧍎
U+274F0 𧓰
U+27753 𧝓
U+281BF 𨆿
U+28206 𨈆
U+282CC 𨋌
U+28617 𨘗
U+28834 𨠴
U+28B4D 𨭍
U+28C3A 𨰺
U+28D63 𨵣
U+29095 𩂕
U+29580 𩖀
U+29BEF 𩯯
U+29FE2 𩿢
U+2A0FF 𪃿
U+2A1CD 𪇍
```

![U+23A15](https://user-images.githubusercontent.com/26766930/156673535-0318ce06-694e-4de3-b6b4-86f549d9ef7e.png)
![U+23E5E](https://user-images.githubusercontent.com/26766930/156673537-d3e36bbb-1b5c-40ad-9176-7b20ac0366eb.png)
![U+23F36](https://user-images.githubusercontent.com/26766930/156673538-10daa7e6-b418-4328-b1da-b7704ba4180e.png)
![U+23FB5](https://user-images.githubusercontent.com/26766930/156673539-a5445061-0a2b-4bfe-9217-66aab9fff75a.png)
![U+2401E](https://user-images.githubusercontent.com/26766930/156673542-20f92d32-edcb-4f97-b177-27bb6ebf6006.png)
![U+2500C](https://user-images.githubusercontent.com/26766930/156673546-d563484a-41d1-48f1-b7ed-7cde6869ec52.png)
![U+2512C](https://user-images.githubusercontent.com/26766930/156673547-a420da08-396d-4c0f-bd90-10be935cf614.png)
![U+251BA](https://user-images.githubusercontent.com/26766930/156673550-a2433934-f435-4933-9b59-5d2b7b5a4165.png)
![U+2526E](https://user-images.githubusercontent.com/26766930/156673552-c613d22c-80b3-448e-9e5c-6d9c7cb727f6.png)
![U+25CC9](https://user-images.githubusercontent.com/26766930/156673554-c6c5671e-7e7f-4690-b346-631fd1bbb2b2.png)
![U+25CD1](https://user-images.githubusercontent.com/26766930/156673556-066b2c75-4ba5-4d3b-aea3-ee129e58bf00.png)
![U+25D87](https://user-images.githubusercontent.com/26766930/156673559-7958f157-90c5-4121-9340-cb8e24dbfa2d.png)
![U+25DFA](https://user-images.githubusercontent.com/26766930/156673560-f85aacaa-f6e6-4106-a578-9ebbfab6218f.png)
![U+25E13](https://user-images.githubusercontent.com/26766930/156673563-ba7805df-bf76-435b-8adb-ae04e70b2748.png)
![U+2734E](https://user-images.githubusercontent.com/26766930/156673565-1ce2122a-cf49-47d7-bb95-122a3063943d.png)
![U+274F0](https://user-images.githubusercontent.com/26766930/156673566-828c8ca1-632d-4eba-9e0b-f6c6a1a03e96.png)
![U+27753](https://user-images.githubusercontent.com/26766930/156673567-ce70cd8f-16d4-4d6c-b2f5-e0a28bdd46b9.png)
![U+281BF](https://user-images.githubusercontent.com/26766930/156673570-e1c280a1-f514-4dc1-868b-efaacd983f31.png)
![U+28206](https://user-images.githubusercontent.com/26766930/156673572-001d8211-0113-49a9-9cb2-3a557b4db577.png)
![U+282CC](https://user-images.githubusercontent.com/26766930/156673573-ed6ab595-60d6-40c5-a267-46309364a0ad.png)
![U+28617](https://user-images.githubusercontent.com/26766930/156673576-ec24e980-522a-4c4a-8623-ec7a14b3dba1.png)
![U+28834](https://user-images.githubusercontent.com/26766930/156673578-b7e12c1d-27b0-466c-ab02-1d2f296e1117.png)
![U+28B4D](https://user-images.githubusercontent.com/26766930/156673579-9fa2f6f5-7d1d-457e-b7ae-eb0ba3696579.png)
![U+28C3A](https://user-images.githubusercontent.com/26766930/156673581-560ddb3d-bf94-4936-9292-47023ea81ae5.png)
![U+28D63](https://user-images.githubusercontent.com/26766930/156673584-5d556904-612e-4d99-a864-c24a77efc21d.png)
![U+29095](https://user-images.githubusercontent.com/26766930/156673585-76121217-2c4b-4f17-95ca-5eb109f46cca.png)
![U+29580](https://user-images.githubusercontent.com/26766930/156673586-03c12f05-6e79-438b-9497-03d5a41da2c7.png)
![U+29BEF](https://user-images.githubusercontent.com/26766930/156673588-07b0a0ad-0804-4b6f-89ce-5f3b91154b85.png)
![U+29FE2](https://user-images.githubusercontent.com/26766930/156673590-23786a12-28a9-46e0-9106-c06cfe0fbbcf.png)
![U+2A0FF](https://user-images.githubusercontent.com/26766930/156673594-d5facab6-6fe6-4342-8f31-b2796f99176c.png)
![U+2A1CD](https://user-images.githubusercontent.com/26766930/156673595-90bd7c43-e446-4397-aca8-e49ec98fd475.png)

**Comment by nomfoundation** [comment]
Thanks for reporting these. A quick look suggests that most of these are long standing issues. However, some, such as U+21E85, may have been introduced when we changed our build tools. Since those are recent regressions, we will prioritize those.

Lee

**Comment by nomfoundation** [comment]
Fixed in 5.0,  @tonton-pixel   please verify and close
